copier/profiles.yml

```diff
@@ -0,0 +1,10 @@
+---
+# profile selection
+template_profile:
+  type: str
+  default: ask
+  help: Select a profile
+  choices:
+    Recommended (our defaults for research software): "recommended"
+    Minimum (bare minimum, no extra features): "minimum"
+    Let me choose (decide each feature yourself): "ask"
```

Comment:
When testing package generation, "Let me choose" option is selected as the default. I suggest to change this to "Recommended".